test/Tests/src/Data/Time/Date_Time_Spec.enso

```diff
@@ -594,7 +596,8 @@ spec_with name create_new_datetime parse_datetime nanoseconds_loss_in_precision=
         offset_2_h = ZoneOffset.ofTotalSeconds 2*3600
         tz = Time_Zone.parse "Europe/Warsaw"
         dst_pending = if name.contains "Javascript" then "Javascript implementation does not support time zones correctly, so the tests for conversion around DST switches would fail and thus are disabled. We may revisit once JS gets better time support, see project Temporal: https://tc39.es/proposal-temporal/docs/ and our issue tracking our integration: https://github.com/enso-org/enso/issues/5384" else
-            if loose_zone_equal "" "" then "Loose Zone conversions are on, skipping the test"
+            if name.contains "Python" then "Python doesn't support time zones correctly" else
+                Nothing
```

Comment:
Setting the `dst_pending` to `Nothing` enables the JavaScript and Python failures.